node_modules/oae-core/comments/comments.html

```diff
@@ -3,7 +3,7 @@
 
 <!-- CONTENT -->
 <div class="comments-widget">
-    <ul id="comments-container" class="media-list"><!-- --></ul>
+    <ul id="comments-container" class="media-list" role="list"><!-- --></ul>
```

Comment:
The new comment box will be a part of the same list. Does that need to be labelled in any way?